content/docs/04-clusters/02-data-center/01-maas/05-install-manage-maas-pcg.md

```diff
@@ -0,0 +1,401 @@
+
+</InfoBox>
+
+
+Palette provides an installer in the form of a Docker container that is temporarily deployed on your laptop, workstation, or jump box. You can use the installer on any linux x86-64 system that has a Docker daemon installed and connectivity to Palette and the MAAS identity endpoint. 
+
+<br />
+
+
+<WarningBox>
+
+The installer does not currently work on running on Apple Silicon.
+
+</WarningBox>
+
+
+## Prerequisites
+
+- Canonical [MAAS installed](https://maas.io/docs/how-to-install-maas), set up, and available in your environment.
+
+
+- A linux computer with a Docker daemon installed and a connection to Palette and the MAAS endpoint. The installer must be invoked on a linux system. We have tested the gateway installation using Ubuntu 20.04 on x86-64.
+
+
+- Private cloud gateway IP requirements: <br /><br /> 
+    
+    - For a single-node gateway, one IP address available in the MaaS subnet for the PCG, or three available IP addresses for a three-node gateway.
+    <br />
+
+    - One IP address available in the MaaS subnet for the Kubernetes api-server endpoint, when deploying a three-node gateway.
+
+
+- Sufficient available IPs within the configured MaaS subnets.
+
+<WarningBox>
+
+By default, the MAAS Kubernetes pac uses a pod classless inter-domain routing (CIDR) range of 192.168.0.0/16. Ensure that the pod CIDR range for any clusters you deploy after setting up the PCG do not overlap with the network used by the bare metal machines that MAAS manages.
+
+</WarningBox>
+
+- Each node in the PCG cluster requires a machine from MAAS in a ready state with the following resources:
+
+    <br />
+
+    - 4 CPUs
+    - 8192 MiB memory 
+    - 60 GiB storage
+
+<WarningBox>
+
+We recommend 100 GiB of storage for PCG nodes, as nodes can run out of 60 GoB of storage with prolonged use.
+
+</WarningBox>
+    
+
+- An active [MAAS API key](https://maas.io/docs/api-authentication-reference) which can be generated in the MAAS web console under **My Preferences** > **API keys**. The following is an example key:
+
+  ``APn53wz232ZwBMxDp5:MHZIbUp3e4DJTjZEKg:mdEv33WAG536MhNC8mIywNLtjcDTnFAQ``
+
+ For details, refer to the MAAS document on [how to add an API key](https://maas.io/docs/how-to-manage-user-accounts#heading--api-key).
+
+  <br />
+
+- The DNS server that the PCG installer will use, must be able to resolve the DNS names of machines that MAAS deploys so it can connect to them. The default setup is to use the MAAS server as the DNS server for any bare metal servers that it deploys. The default MAAS DNS zone is ``.maas``. You can use ``.maas`` or you can use the MAAS web console to create a new DNS zone. When you deploy the PCG and clusters, you can select the desired DNS zone in which DNS name records should be created.
+
+    In the MAAS subnets configuration, you can specify which DNS servers that servers in the MAAS subnet should use. 
```

Comment:
@ritawatson "that servers" --> "those servers"

Reply:
fixed typo